ZenPacks/zenoss/PagerDuty/routers.py

```diff
@@ -64,6 +64,7 @@ def updateAccountSettings(self, apiAccessKey=None, subdomain=None, apiTimeout=No
 
         if not apiTimeout:
             account = models.account.Account(subdomain, apiAccessKey)
+            log.info("A zero value results in a default timeout of 40 seconds")
```

Comment:
indicate what has a zero value or is not being set so that we know how to change this.